### ISSUE TITLE
Can't install on Windows ?

### ISSUE THREAD

**Issue by talvasconcelos** [simbo/gulp-static-site-generator#2]
Hi, i'd love to test your gulp plugin, but i needed to install on windows. Whaen try it says : 

npm ERR! node v4.4.7
npm ERR! npm  v2.15.8
npm ERR! code EBADPLATFORM

npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: gulp-static-site-generator@1.0.2
npm ERR! notsup Valid OS:    darwin,linux
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: x64

Is it supposed to do that or can it be fixed?

**Comment by simbo** [comment]
No, you can not install this on Windows, as it is not defined as valid os in the package.json.

I experienced a lot of issues with node modules on windows, especially when it comes to path and file manipulation. That's why i block windows for all my modules, that make heavy use of path/file actions.

I am working on Mac and Linux only and am not able to test or fix this for windows. (And honestly, i don't want to, because it's windows.) ;)

But feel free to test this for windows and if you like it and changes are necessary, please create a pull request.
If you clone or download this package from Github, remove the `os` property from package.json and copy the package folder to your node_modules, you should be able to use and test it on windows (without any warranty).

**Comment by talvasconcelos** [comment]
Thanks man, i use linux too but, when at work i have a windows machine and i'd like to give it a try. I'll try and follow your steps to make it work.
